src/apis/DataDisplayApi.ts

```diff
@@ -0,0 +1,23 @@
+
+import { User } from '../types/UserTypes';
+
+interface DataDisplayResponse {
+  users: User[];
+}
+
+export const getDataDisplay = async (): Promise<DataDisplayResponse> => {
+  try {
+    const response = await fetch('/data-display', {
+      method: 'GET',
+    });
+
+    if (!response.ok) {
+      throw new Error('Failed to fetch data display');
+    }
+
+    const data = await response.json();
+    return data as DataDisplayResponse;
+  } catch (error) {
```

Comment:
error object should be defined with type:
`error: any`
And we don't know the schema for error. So don't print `error` or `error.message`